test/factories/smart_proxy_factory.rb

```diff
@@ -21,7 +21,8 @@
 
         smart_proxy_feature = proxy.smart_proxy_features.select { |spf| spf.feature_id == v3_feature.id }.first
         smart_proxy_feature.capabilities = plugins
-        smart_proxy_feature.settings = {pulp_url: "https://#{Socket.gethostname}"}
+        smart_proxy_feature.settings = { pulp_url: "https://#{Socket.gethostname}",
+                                         content_app_url: "http://localhost:24816" }
```

Comment:
Slightly similar remark here.

Reply:
this is will just be a string comparison so I think its fine to use `localhost`